backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql

```diff
@@ -0,0 +1,21 @@
+alter table nrfc.submission_detail
+add column if not exists work_safe_bc_number varchar(6) null,
+add column if not exists doing_business_as varchar(120) null,
+add column if not exists client_acronym varchar(8) null,
+add column if not exists TBD_ID_TYPE varchar(8) null,
```

Comment:
If this is not defined yet, don't add it

Reply:
This is a draft. I will put the right names shortly.